src/cli/cloud.js

```diff
@@ -105,9 +105,22 @@ export default ({ commandProcessor, root }) => {
 	});
 
 	commandProcessor.createCommand(cloud, 'login', 'Login to the cloud and store an access token locally', {
+		options: {
+			u: {
+				description: 'your username',
+				alias: 'username',
+				nargs: 1
+			},
+			p: {
+				description: 'your password',
+				alias: 'password',
+				nargs: 1
+			}
+		},
```

Comment:
Since there are now several valid syntaxes, add a `description` block to document them:
- `particle login`: prompt for credentials and log in
- `particle login --username user@example.com --password test`: log in with credentials provided on the command line

Reply:
added an `examples` block in https://github.com/particle-iot/particle-cli/pull/428/commits/3f6f5c0ac54f0055d7deb3291e0198799972b6d7 - that's what you meant right @monkbroc ?

Reply:
Perfect!